app/assets/javascripts/AdverseEvent.js

```diff
@@ -4,6 +4,8 @@ const Code = require('./basetypes/Code');
 const Interval = require('./basetypes/Interval');
 const Quantity = require('./basetypes/Quantity');
 const DateTime = require('./basetypes/DateTime');
+const { ComponentSchema } = require('./Component');
+const { FacilityLocationSchema } = require('./FacilityLocation');
```

Comment:
This type has now been introduced, but isn't being used below to specify `facilityLocation` (line 20)

Reply:
if this is actually an issue, make sure to check this throughout the code.

Reply:
It is okay for these to be included. Component and FacilityLocation are essentially basetypes that may or may not be used